Dockerfile

```diff
@@ -5,6 +5,8 @@ FROM openjdk:11-jdk
 ENV JAVA_HOME=/usr/local/openjdk-11
 ENV PATH="$JAVA_HOME/bin:${PATH}"
 
+ARG TERRAFORM_DIR
+
```

Comment:
Is there some convention on where to put ARGs in Dockerfiles? Perhaps at the top with some very explicit comment on what they mean

Reply:
Added comment